metadata/base/metadata-grpc-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: metadata-grpc-configmap
+data:
```

Comment:
Where are `METADATA_GRPC_SERVICE_HOST` and `METADATA_GRPC_SERVICE_HOST` used?

Shouldn't them be used as the deployment name and --grpc_port value in deployment.yaml?